resources/js/components/live-preview/UpdatesIframe.js

```diff
@@ -9,28 +9,43 @@ export default {
 
             const container = this.$refs.contents;
 
-            if (! container.firstChild) {
-                container.appendChild(iframe);
-                return;
-            }
+            if (container.firstChild) {
+                const existingIFrameSource = new URL(container.firstChild.src);
+                const newIFrameSource = new URL(iframe.src);
+
+                existingIFrameSource.searchParams.delete('live-preview');
+                newIFrameSource.searchParams.delete('live-preview');
+
+                const iFrameSourceIsEqual = existingIFrameSource.toString() === newIFrameSource.toString();
```

Comment:
@Buffalom In what situation would you find the iframe source URL to be different?

Reply:
Ah I think I worked it out. This could happen if you change preview targets.